README.md

```diff
@@ -6,24 +6,40 @@ This project will host the minimum guardrails as part of the GC Cloud Operationa
 
 The purpose of the guardrails is to ensure that departments and agencies are implementing a preliminary baseline set of controls within their cloud-based environments. These minimum guardrails are to be implemented within the GC-specified initial period (e.g. 30 days) upon receipt of an enrollment under the GC Cloud Services Framework Agreement.
 
+## Cloud Usage Profiles 
+
+| Ref # 	| Profile 	| Characteristics 	| Applicable Service Model 	| Connection Type  	| In Scope for SCED? 	|
+|-	|-	|-	|-	|-	|-	|
+| 1 	| Experimentation/Sandbox 	| - Cloud-based services used for experimentation/sandbox<br><br>- No direct system to system network interconnections required with GC data centers 	| IaaS, PaaS, SaaS 	| Type 1 - EIS/IIS 	| No 	|
+| 2 	| Non-sensitive cloud-based services  	| - Cloud-based services hosting non-sensitive GC content<br><br>- No direct system to system network interconnections required with GC data centers 	| IaaS, PaaS, SaaS 	| Type 1 - EIS/IIS 	| Interim - No<br><br>Future - For IaaS/PaaS, use GC-CAP with CCCS virtual NBS when available, based on risk profile 	|
```

Comment:
Might make sense to convert to a list, since markdown doesn't automatically do that for items inside tables
```suggestion
| 2 	| Non-sensitive cloud-based services  	| <ul><li>Cloud-based services hosting non-sensitive GC content</li><li>No direct system to system network interconnections required with GC data centers </li></ul>| IaaS, PaaS, SaaS 	| Type 1 - EIS/IIS 	| Interim - No<br><br>Future - For IaaS/PaaS, use GC-CAP with CCCS virtual NBS when available, based on risk profile 	|
```